src/services/translations/__tests__/translation.service.spec.ts

```diff
@@ -47,6 +47,16 @@ describe("WizdomTranslationService", () => {
         expect('[Translation missing: test]').toEqual(actual);
     });
 
+    it("should return key if it's null, even in developermode", () => {
+        var developermode = {} as IWizdomDeveloperMode;
+        var sut = new WizdomTranslationService({            
+        }, developermode);
+        
+        var actual = sut.translate(null);
+
+        expect(null).toEqual(actual);
+    });
```

Comment:
It would be great with a similar test for empty string to prove the pull request solves both null and empty strings as described in the pullrequest title.

Reply:
Test added